### ISSUE TITLE
Use document style in HTML formatting

### ISSUE THREAD

**Issue by tiarossi** [opensagres/xdocreport#82]
I suggest to extends the list of attributes available for <p> HTML tag to permit specify an style already existent in the template document. Something like <p style="template-style:Standard">text<p>. So the new created paragraph use the selected style.

**Comment by tiarossi** [comment]
Let me try again. The tag is:

```
< p >
```

And the suggestion is:
```
< p style="template-style:Standard ">text< p >
```

**Comment by angelozerr** [comment]
Yes it's a good idea. I just find time to do that.

**Comment by angelozerr** [comment]
I close this issue since PR https://github.com/opensagres/xdocreport/pull/83 fix this issue.
